syntax = "proto3";
import "shared.proto";

package SqlApi;

service Sql {
  // Запуск sql запроса
  rpc ExecuteSQL (SqlRequest) returns (SqlReply);
}

message SqlRequest {
  string sql = 1;  
}

message SqlReply {  
  // hrs::ErrorCodes
  uint64 error_code = 1;
  string error_text = 2;
  // sql::Query::ResultType
  uint32 result_type = 3;
  ProtoShared.DataContainer data = 4;
}
